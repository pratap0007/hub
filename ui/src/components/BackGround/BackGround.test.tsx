import React from 'react';
import renderer from 'react-test-renderer';
import BackGround from './BackGround';

it('renders correctly', () => {
  const tree = renderer.create(<BackGround />).toJSON();
  expect(tree).toMatchSnapshot();
});
