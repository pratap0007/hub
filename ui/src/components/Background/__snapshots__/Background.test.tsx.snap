// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the background component 1`] = `
<div
  className="hub-background-image"
  style={
    Object {
      "backgroundImage": "url(background.jpg)",
    }
  }
>
  <div
    className="pf-c-content"
  >
    <h1
      className="hub-background-heading"
      data-ouia-component-id="OUIA-Generated-Text-1"
      data-ouia-component-type="PF4/Text"
      data-ouia-safe={true}
      data-pf-content={true}
    >
      Welcome to Tekton Hub
    </h1>
    <h3
      className="hub-tag-line"
      data-ouia-component-id="OUIA-Generated-Text-2"
      data-ouia-component-type="PF4/Text"
      data-ouia-safe={true}
      data-pf-content={true}
    >
      Discover, search and share reusable Tasks and Pipelines
    </h3>
  </div>
</div>
`;
